### PR TITLE
Regenerate TLS certs for unit tests

### PR DESCRIPTION
**Pull request by rvl**
### Issue Number

ADP-852

### Overview

- Generate new certificates for the TLS tests, because the current ones have expired.
- Add a script to regenerate those files.

### Comments

Using cardano-sl-x509 turned out to be harder than expected ... doh ... should have just used openssl.

### CODE REVIEW

File: lib/core/test/data/PKIs/cardano-sl-x509.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  tls:
+    ca:
+      organization: "Input Output HK"
+      commonName: "cardano-wallet Self-Signed Root CA"
+      expiryDays: 3650
+
+    server:
+      organization: "Input Output HK"
+      commonName: "cardano-wallet API Server"
+      expiryDays: 365
```
Comment (Anviking) [inline lib/core/test/data/PKIs/cardano-sl-x509.yaml:11]: Not 3650?

File: lib/core/test/unit/Cardano/Wallet/Api/Server/TlsSpec.hs [hunk 2]
```diff
@@ -138,6 +135,51 @@ spec = describe "TLS Client Authentication" $ do
                 , statusMessage = "Upgrade Required"
                 }
 
+--
+-- Test data
```
Comment (Anviking) [inline lib/core/test/unit/Cardano/Wallet/Api/Server/TlsSpec.hs:139]: nice

**Review by Anviking — APPROVED** [review]

File: stack.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 #
 # NOTE: Remember to update the version matrix in README.md when
 # bumping the Cardano version.
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/231870a3b785f569daef0596b3a1657324c9394a/snapshots/cardano-1.25.1-GHC-8.10.4.yaml
+resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/cd67c3f16b6c17afe76fbd3e5cefe3be0ad51147/snapshots/cardano-1.25.1.yaml
```
Comment (Anviking) [inline stack.yaml:6]: https://github.com/input-output-hk/cardano-haskell/pull/43/commits/ca421883a2b435a6dfaa637eb9f967d9cdc7b38d 👌

**Review by paweljakubas — APPROVED** [review]
great

**Pull request merged**